GANDLF/models/dynunet_wrapper.py

```diff
@@ -26,35 +62,26 @@ class dynunet_wrapper(ModelBase):
     def __init__(self, parameters: dict):
         super(dynunet_wrapper, self).__init__(parameters)
 
-        # checking for validation
-        assert (
-            "kernel_size" in parameters["model"]
-        ) == True, "\033[0;31m`kernel_size` key missing in parameters"
-        assert (
-            "strides" in parameters["model"]
-        ) == True, "\033[0;31m`strides` key missing in parameters"
-
-        # defining some defaults
-        # if not ("upsample_kernel_size" in parameters["model"]):
-        #     parameters["model"]["upsample_kernel_size"] = parameters["model"][
-        #         "strides"
-        #     ][1:]
-
+        patch_size = parameters.get("patch_size", None)
+        spacing = parameters.get(
+            "spacing", [1.0 for i in range(parameters["model"]["dimension"])]
```

Comment:
I would suggest using either `"spacing_for_internal_computations"` or `""spacing_for_model_computations"` or something else that suggests this is different from the spacing that we would get from the dataset. This would make it much clearer for the user as well. 

On a separate note, can you add a feature request to calculate the median spacing and size of the training data?

Reply:
I changed the name. I will create a new PR for the calculation of media spacing and size.

Reply:
Sure, but please record the feature request first. 😄